src/plots/xyPlot.ts

```diff
@@ -31,8 +31,8 @@ export class XYPlot<X, Y> extends Plot {
     var _deltaY = 0;
     var _scalingX = 1;
     var _scalingY = 1;
-    var _lastSeenDomainX: X[] = null;
-    var _lastSeenDomainY: Y[] = null;
+    var _lastSeenDomainX: X[] = [null, null];
+    var _lastSeenDomainY: Y[] = [null, null];
```

Comment:
It will not work because `scale.domain()` returns `X[]` and 
```
Type 'X[]' is not assignable to type '[X, X]'.
  Property '0' is missing in type 'X[]'.
```